src/content/getting-started/designers/designers.md

```diff
@@ -6,14 +6,14 @@ title: Designers
 
 <grid-wrapper col_lg="8" flex="true" bleed="true">
 <clickable-tile
-    title="Add Carbon design kit to Sketch library"
+    title="Add Carbon Design Kit to Sketch library"
```

Comment:
Actually we are treating "Carbon Design Kit" as a formal entity and capitalizing it.

Reply:
Ah good, must have misread that somehow. Carry on :)